test/__snapshots__/to-have-class.spec.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`to-have-class fails when the actual does not match the expected 1`] = `
+"expected <div className=\\"root top\\"><span className=\\"child bottom\\">test</span></div>
+to have class 'hotpink'"
+`;
+
+exports[`to-have-class fails when the actual does not match the expected 2`] = `
+"expected <div className=\\"root top\\"><span className=\\"child bottom\\">test</span></div>
+to have class 'rot top'"
```

Comment:
I can't understand why you won't get a contains diff in addition to the standard error message. But I'm planing to go through all of the error messages and see how we can improve the output. Right not it is mainly getting the assertions right that is the focus.